npm/test.js

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env node
+require('shelljs/global');
+require('colors');
+
+require('async').series([
+    require('./test-lint'),
+    require('./test-system'),
+    require('./test-integration'),
+    process.env.CI ? function (done) { done(); } : require('./test-browser')
```

Comment:
I'll be adding the unit test script in a separate pull request to this branch, which will also take care of istanbul test coverage.